compose.yaml

```diff
@@ -1,6 +1,6 @@
 services:
   db:
-    image: postgres:10.19
+    image: postgres:14
```

Comment:
Why this version when we don't support Ubuntu 22? We are currently jumping from 20 to 24.

Reply:
I though Ubuntu 22 was the highest version that was running in prod for now, I am a bit out of the loop. I'll upgrade it to 16 which is the version included in Ubuntu 24